include/CMakeLists.txt

```diff
@@ -9,3 +9,9 @@ if(INSTALL_MBEDTLS_HEADERS)
         PERMISSIONS OWNER_READ OWNER_WRITE GROUP_READ WORLD_READ)
 
 endif(INSTALL_MBEDTLS_HEADERS)
+
+# Make scripts and data files needed for testing available in an
+# out-of-source build.
```

Comment:
Comment says `scripts and data files needed` but it looks like just the headers directory from the function call.